rackcity/utils/change_planner_utils.py

```diff
@@ -359,7 +359,12 @@ def get_changes_on_asset(asset, asset_cp):
     fields = [field.name for field in Asset._meta.fields]
     changes = []
     for field in fields:
-        if getattr(asset, field) != getattr(asset_cp, field):
+        if field == "chassis":
+            chassis_live = asset.chassis
+            chassis_cp = asset_cp.chassis
+            if chassis_live and (chassis_live != chassis_cp.related_asset):
+                changes.append("chassis")
+        elif getattr(asset, field) != getattr(asset_cp, field):
```

Comment:
chassis should be the only field we have to add extra logic to check if they're different on asset + assetcp because asset's chassis is an Asset and assetcp's chassis is an AssetCP

Reply:
lol forgot we already had this helper method... also i think we should also check if an asset is being moved between offline sites?

Reply:
wait jk nvm